### ISSUE TITLE
Myanmar: Glyphs should not look the same

### ISSUE THREAD

**Issue by agguser** [googlefonts/noto-fonts#1738]
### Font
NotoSansMyanmar-Regular.ttf, …
NotoSerifMyanmar-Regular.ttf, …

### Font Version
2.001

### Issue
Some single/combined glyphs look the same, hard to be differentiated. Please make them distinct.

A | B
--- | ---
ဝ | ၀ (0)
ဦ | ဦ (ဥ+ီ)
ဩ | သြ (သ+ြ)
ဪ | သြော် (သ+ြ+ေ+ာ+်)

NotoSansMyanmar-Regular:
![NotoSansMyanmar-Regular-14](https://user-images.githubusercontent.com/1206106/81460557-55630800-91d0-11ea-99b3-4458acc2f946.png)

NotoSerifMyanmar-Regular:
![NotoSerifMyanmar-Regular-14 png](https://user-images.githubusercontent.com/1206106/81460567-601d9d00-91d0-11ea-8a7a-60f01ec5fe15.png)

compare with ZawDecode-Regular:
![ZawDecode-Regular-14](https://user-images.githubusercontent.com/1206106/81460571-63b12400-91d0-11ea-837c-6e58e00fcc60.png)

**Comment by nizarsq** [comment]
I can see the single and combined glyphs looks the same for all characters  mentioned except for ဦ (U+1026), single/combined looks identical to me.
<img width="779" alt="Screen Shot 2020-05-25 at 3 16 56 PM" src="https://user-images.githubusercontent.com/22158882/82845941-692e8f80-9e9b-11ea-8d2f-5e8855ca817e.png">

**Comment by ohbendy** [comment]
If I understand correctly, agguser would like these four sequences to differ in appearance:

- Consonant Wa uni101D ≠ Burmese zero uni1040;
- Independent vowel UU uni1026 ≠ independent vowel U uni1025 + vowel ii uni102E;
- Independent vowel O uni1029 ≠ consonant θa uni101E + medial Ra uni103C;
- Independent vowel Au uni102A ≠ consonant θa uni101E + medial Ra uni103C + vowel e uni1031 + vowel aa uni102C + asat uni103A.

The first of these is fairly common in digital typefaces, though not in handwriting. It is a good idea for Noto to differentiate the Wa from the zero, to avoid the possibility of spoofing.

I've just consulted with a Burmese expert, and we are puzzled by the latter three suggestions. While I understand the principle of making different character sequences look visually distinct, there's no tradition of making these three look different in either handwriting or in digital typefaces. If you have examples where these differ, I'd be very interested to see. I don't consider the Zawdecode solution elegant, it doesn't look like an intentional design decision to me — I think more users would complain about the wraparound having a gap than about distinguishing independent vowels from medial Ra sequences.

I'd also suggest that there's not really a possibility of confusion between the sequences, as the sequences with medial Ra would always have a dependent vowel sign. The independent vowels could never take an independent vowel sign.

**Comment by agguser** [comment]
With different appearance, you can know real characters used by just looking, same benefit as differentiating I/l/1 or O/0. Even when there is no ambiguity, the differentiation let you know typo.

**Comment by ohbendy** [comment]
Yes, I don't disagree. It's just that the script doesn't have a tradition of differentiating the shapes. We can't just invent new forms to make them differ.

**Comment by agguser** [comment]
Well, I don't think that we "must" follow tradition, especially when the change is for the better.

**Comment by ohbendy** [comment]
I'm all for experimentation and new ideas, but if we make glyphs that users are unfamiliar with, that's going to cause readability problems and bug reports. In the case of ZawDecode, I'm not sure how many users would be happy with the disconnected wraparound and whether it's evident which version is O and which version is θra. If you have examples of a better way to disambiguate them, I'd be very interested to see.

The [design brief](https://www.google.com/get/noto/help/faq/) for Noto is not to be experimental, but to 'maintain authenticity' and be 'conservative'.
